docs/EXPLANATION-development-guidelines.md

```diff
@@ -3,8 +3,8 @@
 This document describes tips and current gotchas in our code base and explains the context
 for parts that are evolving in a certain direction (or we *want* to evolve in a certain direction).
 
-See [CONTRIBUTING.md](../CONTRIBUTING.md) and the [README](../README.md) for setup
-instructions and an overview about how to help and join in development.
+See [ENVIRONMENT](https://github.com/percworld/Spoke/blob/main/docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md) to calibrate your development environment, [DEVELOPMENT TIPS](https://github.com/MoveOnOrg/Spoke/blob/main/docs/EXPLANATION-development-guidelines.md), [CONTRIBUTING](../CONTRIBUTING.md) instructions and an overview about how to help and join in development.
```

Comment:
Same comment: Can you link to the main repo instead of your fork? https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md

Reply:
Complete, thank you again.